extensions/azure/data-plane/storage/build.gradle.kts

```diff
@@ -12,32 +12,25 @@
  *
  */
 
-val storageBlobVersion: String by project;
+val storageBlobVersion: String by project
 val jodahFailsafeVersion: String by project
-val jupiterVersion: String by project
-val faker: String by project
 
 plugins {
     `java-library`
-    `java-test-fixtures`
 }
 
 
 dependencies {
-    implementation(project(":extensions:azure:data-plane:common"))
+    api(project(":extensions:data-plane:data-plane-spi"))
+    implementation(project(":extensions:azure:blobstorage:blob-core"))
```

Comment:
below it has `api(project(":extensions:azure:blobstorage:blob-core"))` as well ? Both needed ?